modules/nf-core/pretextmap/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: "pretextmap"
+description: converts bam into genome contact map
+keywords:
+  - genome contact map
+tools:
+  - "pretextmap":
+      description: "Paired REad TEXTure Mapper. Converts SAM formatted read pairs into genome contact maps."
+      homepage: "https://github.com/wtsi-hpag/PretextMap"
+      documentation: "https://github.com/wtsi-hpag/PretextMap/blob/master/README.md"
+      tool_dev_url: "None"
+      doi: ""
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
```

Comment:
Call it `input` and describe the different options

Reply:
done